src/templates.py

```diff
@@ -84,3 +84,25 @@
         background-color: #ffffff18;
     }
 """
+
+# ------------------------------------------------------------
+
+blame_inline_phantom_html_template = """
+    <body id="inline-git-blame">
+        <style>{css}</style>
+        <div class="phantom">
+            <span class="message">
+                {author},&nbsp;{date}&nbsp;{time}
+            </span>
+        </div>
+    </body>
+"""
+
+
+blame_inline_phantom_css = """
+    div.phantom {
+        color: gray;
```

Comment:
For colour, probably the best thing to do is somehow find out what is used for Comments in the user's current colour scheme and then apply some [adjustment](https://www.sublimetext.com/docs/color_schemes.html#colors) to that, like reduced saturation.

Reply:
Sorry, I didn't understand the docs about the adjustment. But I figured I could just invert colors and add alpha. It works as expected, I tested in both color scheme variants.

I must admit, the color inverter is a bit cryptic, but I added a comment for future generations.